src/libraries/System.Reflection.Context/tests/ProjectingFieldInfoTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Reflection.Context.Tests
+{
+    public class ProjectingFieldInfoTests
+    {
+        private readonly CustomReflectionContext _customReflectionContext = new VirtualPropertyInfoCustomReflectionContext();
+        private TypeInfo _customTypeInfo;
+        private FieldInfo _field;
+        public ProjectingFieldInfoTests()
+        {
+            TypeInfo typeInfo = typeof(SecondTestObject).GetTypeInfo();
+            _customTypeInfo = _customReflectionContext.MapType(typeInfo);
+            _field = _customTypeInfo.GetField("field");
+        }
+
+        [Fact]
+        public void DeclaringType_ReturnsCustomType()
+        {
+            Assert.Equal(ProjectionConstants.CustomType, _field.DeclaringType.FullName);
```

Comment:
This is one of the tests that is failing. It looks like DeclaringType is `SecondTestObject`.

Did this succeed for you (locally)?

Reply:
I've got myself setup again, and this is failing. For whatever reason, when i initially PR'd this i thought it was passing.  I'll button up the failures.